docs/MANUAL.rst

```diff
@@ -393,6 +393,14 @@ However, if unset through ``-UNDEBUG``, all ``assert`` are executed and
 eventually raise an ``AssertionError``. Additionnaly, many internal checks are
 done and may fail with a C-ish assertion.
 
+Thread safety
+-------------
+
+By default Pythran dones not generate thread-safe code: reference counting for automatic
+deletion of objects is not done atomically by default. It's possible to force pythran
+to generate thread-safe reference counting by defining the flag ``THREAD_SAFE_REF_COUNT``
+via ``-DTHREAD_SAFE_REF_COUNT``. There is a small performance penalty associated with this.
+
```

Comment:
+ Note that if OpenMP is activated, Pythran automatically switches to  thread-safe reference counting.